packages/orbit-components/src/utils/Grid/Grid.stories.tsx

```diff
@@ -1,100 +1,75 @@
 import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import { SPACINGS_AFTER } from "../../common/consts";
 
 import Grid from ".";
 
-export default {
```

Comment:
Overall, SB for this component is a little bit difficult to understand. 😢  Have you tried different ways how to achieve the goal? 🤔

Reply:
I think now with added default values, extra info and actually working controls it's like any other story, what do you think?